patches.def.h

```diff
@@ -281,6 +281,14 @@
  */
 #define NEWTERM_PATCH 0
 
+/* This patch allows st to fetch the current working directory through the OSC 7 escape
+ * sequence emitted by shells. Must be used with newterm patch.
+ *
+ * https://codeberg.org/dnkl/foot/wiki#spawning-new-terminal-instances-in-the-current-working-directory
+ * https://github.com/veltza/st-sx/commit/817865c2c6ed905af8849580e58bdcf399216fbd
+ */
+#define OSC7_PATCH 0
+
```

Comment:
You accidentally put this in the wrong place. The patches are arranged alphabetically as you know.

Reply:
I changed the location of this toggle.